index.html

```diff
@@ -16,24 +17,51 @@
 
 <body>
 
-  
-     
+  <header>
+    <div class="nav-container">
+      <h2>Fun Bus</h2>
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About Us</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+      </nav>
+    </div>
+  </header>
+
   <div class="container home">
-   
+    
+    <section class="about-section">
+        <img src="/img/fun-bus.jpg" alt="fun bus">
```

Comment:
Be careful with the `src` here. Adding `/` in front of the path means the root of wherever the page is on. I'm not too clear on this myself personally, though you can join me and look at this [reference](https://stackoverflow.com/a/24028813) for clues.

Regardless of the specifics, the `/` renders the image unloadable. Removing the `/` in the src will correctly point to the `fun-bus.jpg` in the `/img` folder that's on the same level as `index.html`.